tests/test_command_telemetry_client.py

```diff
@@ -132,7 +135,7 @@ def connect_callback(self, server):
             )
 
     def config_callback(self, server):
-        self.config_queue.put_nowait(server.config)
+        self.config_list.append(server.config)
```

Comment:
Why do you decide to use the `list` instead? I thought the `async.Queue` is safer?

Reply:
It isn't a question of safety. I was using `asyncio.Queue` because that was the only way I could wait for a new config. Now that I have the new `config_task` attribute I can wait for a config using that mechanism. This is a nice simplification in the unit test because it matches how the telemetry callback works and because I find lists easier to work with than queues.